05_testing_and_ci/examples/python_testing/test_operations_unittests.py

```diff
@@ -0,0 +1,80 @@
+"""
+Tests for mathematical operations functions.
+"""
+from operations import find_max, find_mean
+from unittest import TestCase
+import csv
+
+
+class TestOperations(TestCase):
+    """
+    Test suite for mathematical operations functions.
+    """
+    def setUp(self):
```

Comment:
```suggestion
    def set_up(self):
```
?

Reply:
The function [setUp](https://docs.python.org/3/library/unittest.html#organizing-test-code) is a standard function in `unittest` and hence I need to keep the name the same.